apps/ml/requirements.txt

```diff
@@ -0,0 +1,182 @@
+distlib==0.3.4
+distro==1.5.0
+distro-info==1.0
+Django==4.0
+django-cors-headers==3.10.1
+djangorestframework==3.13.1
+docker==4.4.4
+docker-compose==1.27.4
+dockerpty==0.4.1
+docopt==0.6.2
+duplicity==0.8.20
+entrypoints==0.3
+fasteners==0.14.1
+filelock==3.6.0
+fire==0.4.0
+flatbuffers==2.0
+future==0.18.2
+gast==0.4.0
+google==3.0.0
+google-api-core==1.31.5
+google-api-python-client==1.8.0
+google-auth==1.35.0
+google-auth-httplib2==0.0.3
+google-auth-oauthlib==0.4.1
+google-drive==0.3.2
+google-pasta==0.2.0
+googleapis-common-protos==1.56.0
+grpcio==1.43.0
+Guake==3.6.3
+gyp==0.1
+h5py==3.6.0
+httplib2==0.18.1
+idna==2.8
+importlib-metadata==4.10.0
+ipykernel==4.6.1
+ipython==5.5.0
+ipython-genutils==0.2.0
+jedi==0.18.1
+jeepney==0.7.1
+Jinja2==3.0.3
+jsonschema==4.3.2
+jupyter-client==7.1.0
+jupyter-core==4.9.1
+jupyterlab-pygments==0.1.2
+kaggle==1.5.12
+keras==2.7.0
+Keras-Preprocessing==1.1.2
+keyring==23.0.1
+language-selector==0.1
+launchpadlib==1.10.13
+lazr.restfulclient==0.14.2
+lazr.uri==1.0.5
+libclang==12.0.0
+lockfile==0.12.2
+louis==3.18.0
+macaroonbakery==1.3.1
+Mako==1.1.3
+Markdown==3.3.6
+MarkupSafe==2.0.1
+matplotlib-inline==0.1.3
+mistune==0.8.4
+monotonic==1.5
+more-itertools==4.2.0
+nbclient==0.5.9
+nbconvert==6.3.0
+nbformat==5.1.3
+nest-asyncio==1.5.4
+netifaces==0.10.9
+numpy==1.21.5
+oauth2client==4.1.3
+oauthlib==3.1.0
+olefile==0.46
+opt-einsum==3.3.0
+packaging==21.3
+pandas==1.3.5
+pandocfilters==1.5.0
+paramiko==2.7.2
+parso==0.8.3
+pbr==5.6.0
+pexpect==4.8.0
+pickleshare==0.7.5
+Pillow==8.1.2
+platformdirs==2.5.1
+portpicker==1.2.0
+prompt-toolkit==1.0.18
+protobuf==3.12.4
+ptyprocess==0.7.0
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+pybind11==2.9.0
+pycairo==1.16.2
+pycups==2.0.1
+PyDrive==1.3.1
+Pygments==2.10.0
+PyGObject==3.40.1
+PyJWT==1.7.1
+pymacaroons==0.13.0
+PyNaCl==1.4.0
+pyparsing==3.0.6
+pyRFC3339==1.1
+pyrsistent==0.18.0
+python-apt==2.2.1
+python-dateutil==2.8.1
+python-debian===0.1.39ubuntu1
+python-dotenv==0.15.0
+python-multipart==0.0.5
+python-slugify==5.0.2
+pytz==2021.1
+pyxdg==0.27
+PyYAML==5.3.1
+pyzmq==22.3.0
+reportlab==3.5.66
+requests==2.21.0
+requests-oauthlib==1.3.0
+rsa==4.8
+runipy==0.1.5
+scipy==1.7.3
+SecretStorage==3.3.1
+simplegeneric==0.8.1
+simplejson==3.17.2
+six==1.12.0
+soupsieve==2.3.1
+sqlparse==0.4.2
+systemd-python==234
+tensorboard==2.7.0
+tensorboard-data-server==0.6.1
+tensorboard-plugin-wit==1.8.0
+tensorflow==2.7.0
+tensorflow-estimator==2.7.0
+tensorflow-io-gcs-filesystem==0.23.1
+termcolor==1.1.0
+terminado==0.12.1
+testpath==0.5.0
+text-unidecode==1.3
+texttable==1.6.3
+tornado==4.5.3
+tqdm==4.62.3
+traitlets==5.1.1
+typing_extensions==4.0.1
+ubuntu-advantage-tools==27.7
+ubuntu-drivers-common==0.0.0
+ufw==0.36.1
+unattended-upgrades==0.1
+uritemplate==3.0.1
+urllib3==1.24.3
+usb-creator==0.3.7
+virtualenv==20.14.0
+wadllib==1.3.5
+wcwidth==0.2.5
+webencodings==0.5.1
+websocket-client==0.57.0
+Werkzeug==2.0.2
+wrapt==1.13.3
+xdg==5
+xkit==0.0.0
+zipp==1.0.0
```

Comment:
All of this is not needed for now, just add requirements from here: https://fastapi.tiangolo.com/deployment/docker/

we will extend this with our scikit modules later